src/backend/InvenTree/InvenTree/version.py

```diff
@@ -110,39 +115,40 @@ def inventreeVersionTuple(version=None):
     return [int(g) for g in match.groups()] if match else []
 
 
-def isInvenTreeDevelopmentVersion():
+def isInvenTreeDevelopmentVersion() -> bool:
     """Return True if current InvenTree version is a "development" version."""
     return inventreeVersion().endswith('dev')
 
 
-def inventreeDocsVersion():
+def inventreeDocsVersion() -> str:
     """Return the version string matching the latest documentation.
 
     Development -> "latest"
     Release -> "major.minor.sub" e.g. "0.5.2"
     """
     if isInvenTreeDevelopmentVersion():
         return 'latest'
-    return INVENTREE_SW_VERSION  # pragma: no cover
+    else:
+        return 'stable'
```

Comment:
@SchrodingersGat are you sure this will work as intended?
If someone installs 1.0 and we release bit, the docs will point at eg. 1.2 instead of 1.0